tools/code_format/check_format.py

```diff
@@ -395,7 +395,7 @@ def allow_listed_for_raw_try(self, file_path):
     def deny_listed_for_exceptions(self, file_path):
         # Returns true when it is a non test header file or the file_path is in DENYLIST or
         # it is under tools/testdata subdirectory.
-        return (file_path.endswith('.h') and not file_path.startswith("./test/") and not file_path in self.config.paths["exception"]["include"]) \
+        return (file_path.endswith('.h') and file_path.startswith("./source/") and not file_path.startswith("./source/extensions/")and not file_path in self.config.paths["exception"]["include"]) \
```

Comment:
```suggestion
        return ((file_path.endswith('.h') and file_path.startswith("./source/") and not file_path.startswith("./source/extensions/")and not file_path in self.config.paths["exception"]["include"])
```

Reply:
generally with python its best to avoid using `\` to continue lines - it certainly confused me - the preferred way is to bracket the expresssion - ie 

```python
return (
    some_condition
    or some_other_condition)
```

Reply:
(suggestion above requires fixing next couple of lines also)